tests/utils/cms/cms.go

```diff
@@ -255,6 +255,29 @@ func RetrieveClusterMachinePool(connection *client.Connection, clusterID string,
 	return resp.Body(), nil
 }
 
+func CreateClusterAutoscaler(connection *client.Connection, clusterID string, body *cmv1.ClusterAutoscaler) (*cmv1.AutoscalerPostResponse, error) {
```

Comment:
Will other cases use the CreateClusterAutoscaler by SDK directly?

Reply:
I use it to recover the original clusterautoscaler